src/main/java/frc/robot/subsystems/drive/Drivetrain.java

```diff
@@ -241,6 +241,10 @@ private void move(Vector spdVec, double rot) {
    * @param rateLimit whether or not to use slew rate limiting
    */
   private void move(Vector spdVec, double rot, boolean rateLimit) {
+    Vector spdCommanded = spdVec;
```

Comment:
Why create a second vector?

Reply:
it's not a second vector, it's a reference to the first one, which we shouldn't need, but according to testing we might, though I'll refactor this out if possible in the drivetrain branch

Reply:
What in testing lead you to adding this?